clients/python/llmengine/model.py

```diff
@@ -72,41 +73,43 @@ def create(
                 Source of the LLM. Currently only HuggingFace is supported
 
             inference_framework (`LLMInferenceFramework`):
-                Inference framework for the LLM. Currently only DeepSpeed is supported
+                Inference framework for the LLM. Currently supported frameworks are `deepspeed`, `text_generation_inference`, `vllm`, and `lightllm`
```

Comment:
How would a user make use of `vllm`? Would they pass in `inference_framework=vllm`, or `inference_framework="vllm"`...? It seems like we need to be precise about the literal value, e.g. `LLMInferenceFramework.VLLM`.